.github/workflows/electron-osx.yml

```diff
@@ -0,0 +1,47 @@
+name: Electron Rebuild CI - OSX (x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5
+          - 6
+          - 7
+          - 8
+        include:
+          # includes a new variable for each electron (essentially a switch statement) 
+          - electron: 5
+            nodeVersion: 70
+          - electron: 6
+            nodeVersion: 73
+          - electron: 7
+            nodeVersion: 75
+          - electron: 8
+            nodeVersion: 76
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
+    - name: Install packages
+      run: npm install --unsafe-perm
+    - name: Install electron-rebuild
+      run: npm install --save-dev electron-rebuild@latest
+    - name: Run on ${{ matrix.os }} - Electron v${{ matrix.electron }} (Node ${{ matrix.nodeVersion }})
+      run: ./node_modules/.bin/electron-rebuild -v=${{ matrix.electron }} -w node-sass
```

Comment:
How does `electron-rebuild` really work? What is used to finally kick off the build? `gyp`? Something else?

Reply:
You're right. It's kind of like a wrapper over `node-gpy` (the latest version).

You can get the same result [here](https://github.com/sass/node-sass/blob/499e06685c71f446d8314673ece3617020ab6b88/.github/workflows/electron-linux.yml#L42) with node-gyp but it requires a bump, this tweak doesn't and can be applied to the current release 🎆🎉